integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -38,10 +38,10 @@ def transmit(self, payload, **kwargs):
         """
         Transmit content metadata items to the integrated channel.
         """
-        items_to_create, items_to_update, items_to_delete, transmission_map, content_update_times_by_id = \
-            self._partition_items(payload)
+        items_to_create, items_to_update, items_to_delete, transmission_map, content_update_times_by_id, \
```

Comment:
Any particular reason you decided not to go with the `items_to_skip` idea?

Reply:
How would that work? Skipping an item is determined by its presence within the exporter's payload and in the audit table. `Partition items` already decides what items need to be skipped, the trick here is that we need catalog uuid's associated with each particular content item so that when we create new audit items (in `_transmit_create`) we can add the catalog uuid to the record to be used by future skipped catalogs.

Reply:
hmm so how / where exactly are we detecting that the content has not changed and there can be skipped? sorry I read and may have missed the logic. also the addition of the extra field is auto handled by the diff() we do, no? just need to ensure that.

Reply:
is it more like `Skipping an item is determined by its presence within the exporter's payload and in the audit table.` AND `the content item is the same` ?